.editorconfig

```diff
@@ -0,0 +1,11 @@
+# top-most EditorConfig file
```

Comment:
I use 4 spaces in every other project. To avoid my editor arguing with this project i added an .editorconfig. checkout the site that explains this at: http://editorconfig.org/